extensions/src/poseHand/index.ts

```diff
@@ -0,0 +1,324 @@
+import { ArgumentType, BlockType, Extension, Block, DefineBlock, Environment, ExtensionMenuDisplayDetails, RuntimeEvent } from "$common";
+
+import Video from '../../../packages/scratch-vm/src/io/video';
+import * as handpose from '../../../packages/scratch-vm/node_modules/@tensorflow-models/handpose/dist';
+
+/**
+ * States the video sensing activity can be set to.
+ * @readonly
+ * @enum {string}
+ */
+const VideoState = {
+  /** Video turned off. */
+  OFF: 0,
+
+  /** Video turned on with default y axis mirroring. */
+  ON: 1,
+
+  /** Video turned on without default y axis mirroring. */
+  ON_FLIPPED: 2
+};
+
+/**
+ * Contains the details about the Hand Sensing extension
+ */
+type Details = {
+  name: "Hand Sensing",
+  description: "Sense hand movement with the camera.",
+  iconURL: "pose-hand.png",
+  insetIconURL: "pose-hand-small-3.svg"
+};
+
+/**
+ * Contains descriptions of the blocks of the Hand Sensing extension
+ */
+type Blocks = {
+  goToHandPartBlock(handPart: string, fingerPart: number): void;
+  // these video blocks are present in a few different extensions, perhaps making a file just for these?
+  videoToggleBlock(state: number): void;
+  setVideoTransparencyBlock(transparency: number): void;
+};
+
+
+export default class PoseHand extends Extension<Details, Blocks> {
+  /**
+   * The state of where the hand and its parts are estimated to be
+   */
+  handPoseState;
+
+  /**
+   * Flag to determine if the extension has been installed before
+   * @type {boolean}
+   */
+  firstInstall: boolean;
+
+  /**
+   * The hand model from handpose
+   */
+  _handModel;
```

Comment:
consider renaming. The `_` prefix was used in javascript to mean that this is a "private" or "internal" ([see more](https://stackoverflow.com/questions/4484424/is-the-underscore-prefix-for-property-and-method-names-merely-a-convention)), but now through the beauty of typescript, we can just mark this thing as private!

So we can do:

```
private handModel;
```

And like many of my comments, I recognize this is just what was in the original file!